drivers/ipvlan/ipvlan_endpoint.go

```diff
@@ -76,7 +76,9 @@ func (d *driver) DeleteEndpoint(nid, eid string) error {
 		return fmt.Errorf("endpoint id %q not found", eid)
 	}
 	if link, err := ns.NlHandle().LinkByName(ep.srcName); err == nil {
-		ns.NlHandle().LinkDel(link)
+		if dellinkerr := ns.NlHandle().LinkDel(link); dellinkerr != nil {
+			logrus.Warnf("Failed to delete interface (%s)'s link on endpoint (%s) delete: %v", ep.srcName, ep.id, dellinkerr)
```

Comment:
maybe instead of (%s) you can use %q that will keep the string "quoted"

Reply:
ok